### PR TITLE
Fix Icons

### PR DESCRIPTION
**Pull request by Sherlouk**
wrong symbols were being used for previous event

### CODE REVIEW

File: Resources/Views/Home/_previous-conferences.leaf [hunk 1]
```diff
@@ -14,7 +14,7 @@
             <p>🇬🇧 🇩🇪 🇫🇷 🇮🇳 🇺🇸 🇮🇹 🇸🇳 🇰🇷 🇨🇳 🇿🇦 🇵🇰</p>
           </div>
           <div class="mx-3 mx-md-0">
-            <div class="display-3 text-dark mb-1">22</div>
+            <div class="display-3 text-dark mb-1">#import("speakerCount")</div>
```
Comment (adamrushy) [inline Resources/Views/Home/_previous-conferences.leaf:17]: Nice improvement 👌🏼

File: Sources/App/Features/Home/HomeRouteController.swift [hunk 2]
```diff
@@ -165,14 +165,14 @@ struct HomeRouteController: RouteCollection {
     }
     
     private func getPhase(req: Request, event: Event) throws -> Phase {
-        let isPreviousEvent = event.date <= Date() &&
-            // This is a little trick where we can set the data to anything older than 2015 to hide date and tickets
-            event.date.timeIntervalSince1970 > 1420074000
+        // This is a little trick where we can set the data to anything older than 2015 to hide date and tickets
+        let isntHiddenEvent = event.date.timeIntervalSince1970 > 1420074000
```
Comment (adamrushy) [inline Sources/App/Features/Home/HomeRouteController.swift:169]:
```suggestion
        let isEventHidden = event.date.timeIntervalSince1970 > 1420074000
```
or similar?

File: Sources/App/Features/Home/HomeRouteController.swift [hunk 3]
```diff
@@ -165,14 +165,14 @@ struct HomeRouteController: RouteCollection {
     }
     
     private func getPhase(req: Request, event: Event) throws -> Phase {
-        let isPreviousEvent = event.date <= Date() &&
-            // This is a little trick where we can set the data to anything older than 2015 to hide date and tickets
-            event.date.timeIntervalSince1970 > 1420074000
+        // This is a little trick where we can set the data to anything older than 2015 to hide date and tickets
```
Comment (adamrushy) [inline Sources/App/Features/Home/HomeRouteController.swift:168]: nitpick 👀 
```suggestion
        // This is a little trick where we can set the date to anything older than 2015 to hide date and tickets
```

File: Sources/App/Features/Home/HomeRouteController.swift (hunk 2 above)
Comment (Sherlouk) [reply Sources/App/Features/Home/HomeRouteController.swift:169]: It's literally the inverse, but I'll tweak so it reads better 😂

**Review by adamrushy — APPROVED** [review]

**Pull request merged**